Field.php

```diff
@@ -61,6 +63,11 @@ public function setField($_x, $_y, $_state)
         }
     }
 
+    public function field($_x, $_y)
```

Comment:
Having a method that has the same name as the class that it is contained in, looks a little bit strange.
This also applies to the method `setField()`. What about naming these `setFieldValue()` and `fieldValue()`.
If you use PhpStorms Refactoring-Rename feature PhpStorm will automatically adapt all usages of these methods.

Reply:
Also this method has no documentation, so please add one.